mycroft/configuration/config.py

```diff
@@ -183,30 +184,72 @@ def get(configs=None, cache=True):
         Args:
             configs (list): List of configuration dicts
             cache (boolean): True if the result should be cached
+            remote (boolean): False if the Mycroft Home settings shouldn't
```

Comment:
nitpick but I'd prefer we keep this generic - it may be another backend that isn't "Mycroft Home". I also think we're likely to rename "Home" given it can refer to sooooo many things.

can we just use "remote"?

Reply:
Sure we can, that seems fine

Reply:
Updated